### PR TITLE
TicTacToe Testing

### PR DESCRIPTION
**Pull request by OlukaDenis**
- Testing the game functionalities

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import Gameboard from '../src/modules/board';
+import Player from '../src/modules/player';
```
Comment (bolah2009) [inline test/index.test.js:2]: [Optional]

Why not separate the tests into two files? This way you have a test file for `Gameboard` and `Player` separately.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
Awesome job with the tests!

Let's test all the public methods of each function used (`Gameboard` and `Player`).
   -   Each test examples should be testing the public method of `Gameboard`  (`checkWinner`, `getBoard`, `turn` and `reset`) and `Player` - (`name`, `marker`, `getMarker`, `setTurn`, `getTurn`, `getWins`, `setWin`)

#### :recycle:  Make these changes and submit another code review request. 👍🏾

#### Feel free to reach me on slack if you having issues implementing these features. :relaxed: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Comment by kalavhan** [comment]
Hello @bolah2009, thank's a lot for the review, the test for the method getMarker() was missing, is now added and we modified the description of each test to be more descriptive.
Have a nice day!

**Review by bolah2009 — APPROVED** [review]
It's a 👍🏾 from me!

### :heavy_check_mark: Approved!

Good luck with your next projects. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Pull request merged**